src/layout/Tabs/Tab.tsx

```diff
@@ -37,6 +42,13 @@ export interface TabProps extends PropsWithChildren {
    */
   onSelect?: () => void;
 
+  /**
+   * The variant of the tabs.
+   *
+   * @private
+   */
+  variant?: 'primary' | 'secondary' | 'tertiary';
```

Comment:
i think `tertiary` styling hasn't been included. should this variant be here?

Reply:
oops shoulda been removed thanks